site/docs/components/date-picker/usage.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Using the component
+
+### When to use
+
+Use a Date Picker when enabling users to filter data that’s displayed on screen, or whenever a specific timeframe or period needs to be selected.
+
+### Best practices
+
+To improve consistency and simplify your messaging, consider recommending just one acceptable date format in the label and/or tooltip – ideally, the one most frequently used in your application.
```

Comment:
the new approach will be to show the error in the panel header instead of tooltip, so will update this

https://www.figma.com/design/iVb153RCmU5R7uvZ3ySPoR/Date-Picker?node-id=1958%3A77880&t=cSQYF2zmLzvezqKK-1